pkgs/development/python-modules/scrapy/default.nix

```
@@ -49,15 +22,13 @@ buildPythonPackage rec {
     # fix test_get_func_args. remove on next update
     (fetchpatch {
       name = "test_get_func_args.patch";
-      url = "https://github.com/scrapy/scrapy/commit/b1fe97dc6c8509d58b29c61cf7801eeee1b409a9.patch";
+      url =
+        "https://github.com/scrapy/scrapy/commit/b1fe97dc6c8509d58b29c61cf7801eeee1b409a9.patch";
```

Please revert this. `nixfmt` would probably to it in the future.

```suggestion
      url = "https://github.com/scrapy/scrapy/commit/b1fe97dc6c8509d58b29c61cf7801eeee1b409a9.patch";
```

---

Fixed